docs/src/DataStructures/Secp256k1/Response-LoginOnly.md

```diff
@@ -0,0 +1,24 @@
+```json
+{
+  "userPublicKey": {
+    "encodedValue": "f6d1YDa4agkaQ5Kqq8ZKwCf2Ew8UFz9ot2JNrBwHsFkhdtHEn",
+    "encoding": "base58",
+    "format": "ss58",
```

Comment:
I think this should not be our universal account system, but the Ethereum Address setup.